### ISSUE TITLE
Feature Request: Check if device has touchID or faceID

### ISSUE THREAD

**Issue by narek-sv** [dennisweissmann/DeviceKit#139]

**Comment by dennisweissmann** [comment]
Do you have any insights in how this could be done?

The comment under [this answer](https://stackoverflow.com/a/47046245/1489885) suggests that you can only find out if it Face ID is enrolled. It seems there's no possibility to detect the *hardware* capability of the phone, is that right?

**Comment by narek-sv** [comment]
Yes, i think the only way is to create array of all known touchID and faceID devices, then check if it is subset or not, something like this 

```
extension Device {
    static var allTouchID: [Device] {
        return [.iPhone5s, .iPhoneSE, .iPhone6, .iPhone6Plus, .iPhone6s, .iPhone6sPlus, .iPhone7,
                    .iPhone7Plus, .iPhone8, .iPhone8Plus, .iPadAir2, .iPadMini3, .iPadMini4, .iPadPro12Inch,
                    .iPadPro12Inch2, .iPadPro9Inch, .iPadPro10Inch, .iPad5, .iPad6]
    }
    
    static var allFaceID: [Device] {
        return [.iPhoneX]
    }
    
    var hasTouchID: Bool {
        return isOneOf(Device.allTouchID)
    }
    
    var hasFaceID: Bool {
        return isOneOf(Device.allFaceID)
    }
    
    var hasBiometryScanner: Bool {
        return hasFaceID || hasTouchID
    }
}
```

**Comment by dennisweissmann** [comment]
Okay, yeah that makes sense! (a PR is welcome, code LGTM :)) otherwise I'll try to put it in this week

**Comment by dennisweissmann** [comment]
On second thought, I think we should introduce an enum in the Device python class (or two additional properties), so that we don't forget to update those if new devices are added.

**Comment by dennisweissmann** [comment]
Solved via 8627f45ecf000bca74da4f9f90a8651898037691

Thanks for the suggestion @narek-sv!
